src/rustllvm/RustWrapper.cpp

```diff
@@ -545,6 +549,18 @@ static unsigned fromRust(LLVMRustDIFlags Flags) {
     Result |= DINode::DIFlags::FlagRValueReference;
   }
 #if LLVM_RUSTLLVM || LLVM_VERSION_GE(4, 0)
```

Comment:
> i'm not sure why test fails on travis. 

The CI uses the system LLVM 3.9 for testing, so both flags are going to be false, meaning these flags won't be applied at all.

You may move these flags into another `#if` block which does not check `LLVM_RUSTLLVM` (assume these can apply to our Rust-modified LLVM), or tag the test case with `// no-system-llvm`.